lib/OpenQA/Script/CloneJobSUSE.pm

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018-2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package OpenQA::Script::CloneJobSUSE;
+
+use Mojo::Base -strict, -signatures;
+
+use Data::Dump 'pp';
+use Exporter 'import';
+use LWP::UserAgent;
+
+
+our @EXPORT = qw(
+  collect_incident_repos
+  detect_maintenance_update
+);
+
+sub collect_incident_repos ($settings) {
+    my @incident_urls;
+    my $ua = LWP::UserAgent->new;
```

Comment:
It would be nice to re-use the existing `$ua` here (instead of creating a new one every time this point is reached), even though it means passing down the object a few functions down. Then you also don't need to care about setting the timeout here.

Reply:
I Modified the code as suggested. Thanks.